src/docs/asciidoc/udp-server.adoc

```diff
@@ -326,3 +326,70 @@ public class Application {
 }
 ----
 ====
+
+== Metrics
+UDP server supports built-in integration with https://micrometer.io/[`Micrometer`].
+It exposes all metrics with prefix `reactor.netty.udp.server`.
+
+The following table provides information for the UDP server metrics:
+
+[width="100%",options="header"]
+|=======
+| metric name | type | description
+| reactor.netty.udp.server.data.received | DistributionSummary | Amount of the data that is received, in bytes
+| reactor.netty.udp.server.data.sent | DistributionSummary | Amount of the data that is sent, in bytes
+| reactor.netty.udp.server.errors | Counter | Number of the errors that are occurred
+|=======
+
+These additional metrics also will be available:
+
+include::alloc-metrics.adoc[]
+
+The following example enables that integration:
+
+====
+[source,java]
+----
+import reactor.netty.Connection;
+import reactor.netty.udp.UdpServer;
+
+public class Application {
+
+    public static void main(String[] args) {
+        Connection server =
+                UdpServer.create()
+                         .metrics(true) <1>
+                         .bindNow(Duration.ofSeconds(30));
+
+        server.onDispose()
+              .block();
+    }
+}
+----
+<1> Enables the built-in integration with Micrometer
+====
+
+When UDP server metrics are needed, but with an integration with a system different than `Micrometer` or you want
```

Comment:
Change it to the following: When HTTP client metrics are needed for an integration with a system other than Micrometer or you want
to provide your own integration with Micrometer, you can provide your own metrics recorder, as follows:

(Same comment as before.)